static/app/components/badge/groupPriority.tsx

```diff
@@ -285,7 +285,9 @@ const StyledTag = styled(Tag)`
   }
 
   & > div {
```

Comment:
just a nit; any chance there's a better target for this or should it be a separate component?

Reply:
This probably should be cleaned up but it seems relatively safe for now, would rather keep it self-contained than expose another API surface for styling.